persistence/core/src/main/scala/com/lightbend/lagom/internal/persistence/testkit/package.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence
+
+import com.typesafe.config.{ Config, ConfigFactory }
+import play.api.{ Configuration, Environment }
+
+package object testkit {
+  private[lagom] lazy val clusterConfig: Config = ConfigFactory.parseString(
+    s"""
+    akka.actor.provider = akka.cluster.ClusterActorRefProvider
+    akka.remote.netty.tcp.port = 0
+    akka.remote.netty.tcp.hostname = 127.0.0.1
+    """
+  )
+
+  private[lagom] def loadTestConfig(): Config = Configuration.load(Environment.simple()).underlying
+}
```

Comment:
I find these names very confusing. Looking at how it's used:

```scala
testkit.clusterConfig.withFallback(testkit.loadTestConfig())
```

What is being used as a fallback: (1) a `testConfig` I am loading or (2) the _cooked_ `Config` of `loadTests`?

Reply:
Both names are confusing, or just `loadTestConfig`? Should I change the latter to `loadConfig`, or any other suggestions?

Reply:
Hmmm, let's see.

`testkit.clusterConfig` is a hadrcoded config so clustering can be used in tests. I guess `testkit.clusterConfigOverrides` covers the intent but is rather long.

`testkit.loadTestConfig()`, OTOH, loads the config every time, using regular mechanisms, and, to me, the confusion was wrt to `load tests`. In this case I think `testkit.loadConfig()` makes it clear that we are loading a config. Or `testkit.loadConfigForTests()`? Meh, it's not any better. 😒

Reply:
Yeah, this is why I want to keep these private for now... it's all a bit confusing how configuration layers in tests. I guess `testkit.loadConfig()` is the best option for now.

Reply:
On closer look, I'm not even sure why the `clusterConfig` override is necessary at all. The tests pass without it.  I'm going to revert.